src/findings/finding_container.py

```diff
@@ -49,6 +49,27 @@ def getAllFindings(cls):
     def getActionableFindings(cls):
         return [finding for finding in cls._finding_results if finding.actionable]
 
+    @classmethod
+    def toHumanReadableMessage(cls):
+        output_message = ""
+        file_to_findings = {}
+        for finding in cls.getActionableFindings():
+            # Create a map to summarize the findings based on proto file name.
+            if finding.location.proto_file_name not in file_to_findings:
+                file_to_findings[finding.location.proto_file_name] = []
+            file_to_findings[finding.location.proto_file_name].append(finding)
+        # Add each finding to the output message.
+        for file_name in file_to_findings:
```

Comment:
We can save the lookup in the dict by iterating over `items` instead.
```python
for file_name, findings in file_to_findings.items():
```

Reply:
Done, much cleaner now. Thanks!